### ISSUE TITLE
Enabe AF confirmation for manual lens

### ISSUE THREAD

**Issue by GoogleCodeExporter** [400plus/400plus#85]
```
When some manual (i.e. M42) lenses are attached (via adapter) AF-confirmation 
in body is disabled. 
You have to buy special chip ("dandelion") to cheat the body.
It would be nice to have it enabled in firmware (i.e. when you use more manual 
lenses with different mounts etc..)
```

Original issue reported on code.google.com by `dgo...@gmail.com` on 6 Jun 2011 at 9:08
- Blocking: #309
- Blocked on: #117
